docs/presigned-put-upload-via-browser.md

```diff
@@ -1,25 +1,34 @@
-# Upload files from browser using pre-signed URLs [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+# Upload Files from a Browser Using Pre-signed URLs [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
Do we need to say a browser. Because it could be any simple js app they wrote.

Reply:
@deekoder I have made this change in the branch. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-7/docs/presigned-put-upload-via-browser.md.